.github/workflows/muna-frontend.yml

```diff
@@ -0,0 +1,52 @@
+name: CI build
+
+on:
+  push:
+    branches:
+      - automation-test
```

Comment:
should we add master here?

Reply:
Yes

Reply:
please do so. also, please merge this into test.yml